CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- [Added create wallet feature for passkey provider](https://github.com/multiversx/mx-sdk-dapp/pull/1258)
```

Comment:
Why is this the same entry as the one below?

Reply:
fixed